### PR TITLE
Add kubernetes security policy related resources and data sources

### PR DESCRIPTION
**Pull request by schirevko**

### CODE REVIEW

File: examples/kubernetes/security_policy/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      {
+        "min_replicas" = 1
+        "max_replicas" = 2
+      }
+    ]
+    "excludedNamespaces" = []
+  }
+}
+
+resource "vkcs_kubernetes_security_policy" "sch-policy" {
```
Comment (ftersin) [inline examples/kubernetes/security_policy/main.tf:13]: give more real name to the policy

File: vkcs/kubernetes/resource_cluster.go [hunk 2]
```diff
@@ -260,6 +260,12 @@ func ResourceKubernetesCluster() *schema.Resource {
 				Computed:    true,
 				Description: "Custom DNS cluster domain. Changing this creates a new cluster.",
 			},
+			"sync_security_policy": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Enables syncing of security policies of cluster.",
```
Comment (ftersin) [inline vkcs/kubernetes/resource_cluster.go:267]: lets mention default value?

File: examples/kubernetes/security_policy/import.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+terraform import vkcs_kubernetes_security_policy.sp 723bfe25-5b2b-4410-aba0-1c0ef6d1c8b0
```
Comment (ftersin) [inline examples/kubernetes/security_policy/import.sh:1]: move this file to templates dir

File: vkcs/kubernetes/data_source_cluster.go [hunk 4]
```diff
@@ -220,6 +225,7 @@ func dataSourceKubernetesClusterRead(ctx context.Context, d *schema.ResourceData
 	d.Set("loadbalancer_subnet_id", c.LoadbalancerSubnetID)
 	d.Set("insecure_registries", c.InsecureRegistries)
 	d.Set("dns_domain", c.DNSDomain)
+	d.Set("sync_security_policy", c.SecurityPolicySyncEnabled)
```
Comment (paaanic) [inline vkcs/kubernetes/data_source_cluster.go:228]: I don't remember, is it OK to set boolean from nil in SDK?

File: vkcs/kubernetes/resource_security_policy.go [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+		return
+	}
+
+	id := data.ID.ValueString()
+
+	err = securitypolicies.Delete(containerInfraClient, id).ExtractErr()
+	if errutil.IsNotFound(err) {
+		return
+	} else if err != nil {
+		resp.Diagnostics.AddError("Unable to delete resource vkcs_kubernetes_security_policy", err.Error())
```
Comment (paaanic) [inline vkcs/kubernetes/resource_security_policy.go:307]:
```suggestion
		resp.Diagnostics.AddError("Error deleting vkcs_kubernetes_security_policy", err.Error())
```

File: vkcs/kubernetes/resource_security_policy.go [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+	client, err := r.config.ContainerInfraV1Client(region)
+	if err != nil {
+		resp.Diagnostics.AddError("Error creating Kubernetes API client", err.Error())
+		return
+	}
+
+	id := data.ID.ValueString()
+	ctx = tflog.SetField(ctx, "security_policy_id", id)
+
+	staticRouteUpdateOpts := securitypolicies.UpdateOpts{
```
Comment (paaanic) [inline vkcs/kubernetes/resource_security_policy.go:256]:
```suggestion
	updateOpts := securitypolicies.UpdateOpts{
```

File: vkcs/kubernetes/resource_cluster.go (hunk 2 above)
Comment (schirevko) [reply vkcs/kubernetes/resource_cluster.go]: Added

File: examples/kubernetes/security_policy/import.sh (hunk 3 above)
Comment (schirevko) [reply examples/kubernetes/security_policy/import.sh]: Moved

File: vkcs/kubernetes/data_source_cluster.go (hunk 4 above)
Comment (schirevko) [reply vkcs/kubernetes/data_source_cluster.go:228]: Yes, it's ok: https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/resource_data.go#L218

File: vkcs/kubernetes/data_source_security_polity_template.go [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+			},
+			"id": schema.StringAttribute{
+				Computed:    true,
+				Description: "ID of the resource.",
+			},
+			"name": schema.StringAttribute{
+				Required:    true,
+				Description: "Name of the security policy template.",
+			},
+
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/kubernetes/data_source_security_polity_template.go:59]: Added

File: vkcs/kubernetes/resource_security_policy.go (hunk 6 above)
Comment (schirevko) [reply vkcs/kubernetes/resource_security_policy.go]: Renamed

**Review by paaanic — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,6 +6,11 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+#### v0.6.2
```
Comment (ftersin) [inline CHANGELOG.md:9]: 0.7.0

File: vkcs/kubernetes/resource_security_policy.go [hunk 9]
```diff
@@ -0,0 +1,310 @@
...
+				Description: "Policy settings.",
+			},
+
+			"namespace": schema.StringAttribute{
+				Required:    true,
+				Description: "Namespace to apply security policy to.",
+			},
+
+			"enabled": schema.BoolAttribute{
+				Required:    true,
```
Comment (ftersin) [inline vkcs/kubernetes/resource_security_policy.go:97]: what is a reason to do not have default value for the argument?

File: CHANGELOG.md (hunk 8 above)
Comment (schirevko) [reply CHANGELOG.md]: Fixed

**Review by ftersin — APPROVED** [review]

**Pull request merged**